sdk/src/main/scala/com/horizen/SidechainHistory.scala

```diff
@@ -145,7 +145,7 @@ class SidechainHistory private (val storage: SidechainHistoryStorage,
       SidechainBlockInfo.mainchainHeaderHashesFromBlock(block),
       SidechainBlockInfo.mainchainReferenceDataHeaderHashesFromBlock(block),
       WithdrawalEpochUtils.getWithdrawalEpochInfo(block, parentBlockInfo.withdrawalEpochInfo, params),
-      vrfProofHash,
+      vrfOutput.getOrElse(new VrfOutput(Array())), //technically block is not correct from consensus point of view if vrfOutput is None
```

Comment:
Do we use SidechainBlockInfo for verified block now or only after it was verified, so we can relay that proof is valid, so output exists? maybe we can store the output as Option or throw an exception directly here.